addons/event_sale/wizard/event_edit_registration.py

```diff
@@ -46,7 +55,11 @@ def default_get(self, fields):
                     'phone': so_line.order_partner_id.phone,
                     'mobile': so_line.order_partner_id.mobile,
                 }])
+
+        # sort attendees according to their related event and event tickets
+        attendee_list.sort(key=lambda x: x[2]['event_id']*10 + x[2]['event_ticket_id'])
```

Comment:
How about setting a correct "_order" property on the "registration.editor.line" model?
Would be cleaner, this line is a bit cryptic :D

Reply:
`_order` added to `registration.editor.line`! so we can remove this line.